### DISCUSSION TITLE
Just taking advice

### DISCUSSION THREAD

**Discussion by spyke7** [headwaymaps/headway#160] [General]
How you made your project go popular and in which platform you promoted it and how you attracted others?

**Comment by ellenhp** [comment]
Hey! So this is probably not a very helpful answer but I don't really know how this got popular. Most of my projects are things that nobody cares about, but this one made a lot of people really excited. I've posted it on HN twice, and also on Reddit. The main thing that contributed to its "success", I think, is that it solves a problem that people wanted someone to solve.

**Reply by spyke7** [reply]
👌

**Reply by spyke7** [reply]
By HN do u mean hacker news
